test/controllers/ContinuedIllnessController.test.ts

```diff
@@ -0,0 +1,92 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import { createApp } from '../ApplicationFactory';
+
+import { Appeal } from 'app/models/Appeal';
+import { ApplicationData } from 'app/models/ApplicationData';
+import { Illness } from 'app/models/Illness';
+import { Reasons } from 'app/models/Reasons';
+import { CONTINUED_ILLNESS_PAGE_URI } from 'app/utils/Paths';
+
+describe('ContinuedIllnessController', () => {
+
+    it('should show hint containing illness start date, and radio buttons for Yes and No on GET', async () => {
+        const  applicationData: Partial<ApplicationData> = {
+            appeal: {
+                penaltyIdentifier: {
+                    companyNumber: 'NI000000'
+                },
+                reasons: {
+                    illness: {
+                        illnessStart: '2020-5-1'
```

Comment:
I'd expect this to be correct ISO date in YYYY-MM-DD format. Otherwise that would likely need to be reformatted on submission to Appeals API. Can you ask @n-r-b to make sure we store correct ISO string?

Reply:
Noted and changed. I'll confirm with Natalie tomorrow.